Yoast/wordpress-seo
# Add yoast_head property to relevant core REST routes

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Adds to `yoast_head` property to existing REST routes that contains the full Yoast SEO head.

## Relevant technical choices:

* The fields are implemented as a route as they are expected to behave exactly the same. They simply register fields instead of routes.

## Test instructions

This PR can be tested by following these steps:

* Visit `/wp-json/wp/v2/posts`. Every object should have a `yoast_head` property that matches what's output on the front-end for that specific post.
* Do the same for `/wp-json/wp/v2/tags`, `/wp-json/wp/v2/pages`, `/wp-json/wp/v2/users` and `/wp-json/wp/v2/types`. The latter being post type archives.

## UI changes

## Documentation

## Quality assurance


Fixes #14889
